### ISSUE TITLE
Variant should consider de novos when determining compound het inheritiance

### ISSUE THREAD

**Issue by tonydisera** [iobio/gene.iobio.vue#507]
Looking at the slivar compound hets in Mosaic variant sets that Al is preparing for a UDN case, many of the variants did not pass the gene.iobio compound het filter b/c gene.iobio failed to consider a de novos.  For example, if one high impact variant is maternally inherited and the other is de novo, that forms a compound het.

<img width="1172" alt="Screen Shot 2020-05-08 at 10 32 36 AM" src="https://user-images.githubusercontent.com/8420103/81427128-87646380-9117-11ea-9ba1-a5cf1d4ac58a.png">

For example, MAGEC1 has a de novo and when paired with a paternally inherited variant, should be considered a compound het.

<img width="202" alt="Screen Shot 2020-05-08 at 10 35 07 AM" src="https://user-images.githubusercontent.com/8420103/81427244-c2669700-9117-11ea-8679-dc9f25bbf511.png">
<img width="190" alt="Screen Shot 2020-05-08 at 10 35 18 AM" src="https://user-images.githubusercontent.com/8420103/81427267-ce525900-9117-11ea-893a-f3d9222ac677.png">

**Comment by AlistairNWard** [comment]
Do we know that the de novo is not also on the paternally inherited chromosome?

**Comment by tonydisera** [comment]
Good catch, @AlistairNWard!  Yes, some of these slivar variants filtered as compound hets are indeed on chrX.  This requires further discussion.

**Comment by mvelinder** [comment]
Nice find @tonydisera ! 

From the screenshots I see, I don't see any evidence of the de novo being paternally inherited. Not sure what we we mean there.

Agreed this required further discussion, first based on proband sex, and then fairly simple logic thereafter.

**Comment by AlistairNWard** [comment]
The de novo obviously isn't inherited. The point was that to be a compound het, the de novo mutation needs to be on the same chromosome as the paternally inherited variant. We would only know this is a compound het by seeing reads that span both events and show both alt alleles on the same reads.

**Comment by AlistairNWard** [comment]
This is something we should look into, but isn't high priority
